### DISCUSSION TITLE
SBS

### DISCUSSION THREAD

**Discussion by AVGCFan1995** [iptv-org/community#959] [Show and tell]
### Channel ID

SBS.kr

### Stream URL (optional)

http://221.157.125.239:1935/live/psike/playlist.m3u8

### Notes (optional)

I found this link from https://github.com/wcb1969/iptv/blob/main/%E9%9F%A9%E5%9B%BD.txt but this link is dead someone will either repaired it or replaced it with a new one because the SBS TV livestream feed is dead. I hope someone will fix it.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://221.157.125.239:1935/live/psike/playlist.m3u8 (offline)

![image](https://github.com/user-attachments/assets/a54a7dde-9169-46c0-9c95-1e3673ac7329)
